docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+|`GMP work packet processing` subincrement| work packet processing (WPP) operations | concurrent and sometimes final operations during a *STW* to complete the subincrement | `<concurrent-start>`, `<concurrent-end>`| The `GMP work packet processing subincrement` runs immediately after the `global mark phase` subincrement |
+|final global mark phase increment | final global mark phase operations including class unload | *stop-the-world* | `gc-start>`, `<gc-end>`| Final increment. Runs the final global mark phase operations followed by operations to finish the cycle  |
+
+The `balanced` global *mark* GC cycle follows a general structure in the verbose GC log as shown. The lines are indented to help illustrate the flow and some child elements are omitted for clarity:
+
+```xml
+<exclusive-start/>                        (STW pause starts)
+
+<allocation-taxation/>                    (memory threshold trigger recorded)
+
+<cycle-start type="global mark phase"/>   (global mark cycle starts)
+
+<gc-start type="global mark phase"/>      (1st GMP STW subincrement starts)
+
+    <mem-info>                            (memory status before operations)
+
+    <remembered-set>
+
+    </mem-info>         
+
+</gc-start> 
+
+<gc-op type="mark increment" />           (STW copy forward operation completed)
+
+<gc-op  type="class unload" />            (optional STW class unload operation completed)
+
+<gc-end>                                  (partial cycle increment ends)
+
+    <mem-info>                            (memory status after operations)
+          
+    <remembered-set>
+
+    </mem-info>
+
+<gc-end>
+
+<concurrent-start type="GMP work packet processing"/> (1st GMP concurrent subincrement starts)
+
+<exclusive-end/>                                      (STW pause ends)
+
+<concurrent-end type="GMP work packet processing"/>   (1st GMP concurrent subincrement ends)
+
+<gc-op type="mark increment"/>                        (copy forward operation runs concurrently)
+
+</concurrent-end type="GMP work packet processing"/>>
+
+...                                       (partial GC cycles run during STW pauses)
+
+<exclusive-start/>                        (STW pause starts)
+
+<gc-start type="global mark phase"/>                  (2nd STW GMP subincrement starts)
+
+...   
+
+<concurrent-start type="GMP work packet processing"/> (2nd concurrent GMP subincrement starts)
+
+...
+
+<exclusive-end/>
+
+...                                       (more partial GC cycles may run)
+
+<concurrent-end type="GMP work packet processing" />  (2nd concurrent GMP subincrement ends)
+...
+
+</concurrent-end>
+
+...                                       (more partial cycles and GMP increments interleave)
+
+<exclusive-start/>                        (STW pause starts)
+
+...
+
+<gc-start type="global mark phase"/>      (final STW GMP subincrement starts.)
+
+...   
```

Comment:
I would add here gc-op 'mark increment' (it also always exists in the first STW increment of the cycle)
This gc-op will do weak roots processing.
There is also optional gc-op class unloading (this has been already mentioned in the table above).

Reply:
I'll take a look. I'm conscious of putting too much detail in this overview illustration, but will definitely see if that could go in there without the illustration becoming too detailed.